wolfcrypt/src/random.c

```diff
@@ -647,14 +647,7 @@ static int Hash_DRBG_Generate(DRBG_internal* drbg, byte* out, word32 outSz)
         return DRBG_NEED_RESEED;
     }
     else {
-    #ifndef WOLFSSL_SMALL_STACK
         byte digest[WC_SHA256_DIGEST_SIZE];
-    #else
-        byte* digest = (byte*)XMALLOC(WC_SHA256_DIGEST_SIZE, drbg->heap,
```

Comment:
Why remove this?

Reply:
Removed to avoid conention/overhead around heap ops in each generator call.  It's just 32 bytes and no need to keep off stack.  (Done in consultation with @SparkiDev .)

Reply:
I think this smallstack option should not be removed. There is macros like WOLFSSL_SMALL_STACK_CACHE used with the RNG for hanging on to the SHA256 struck. Honestly having the whole wc_Sha256 struct on the stack already is going over the goal of 100 bytes or less of stack.